pkg/validation/validation.go

```diff
@@ -139,23 +163,24 @@ func (v *validator) IsValidUpgradeConfig(c client.Client, uC *upgradev1alpha1.Up
 				Message:           err.Error(),
 			}, err
 		}
-		versionValid, versionAvailable, err := versionValidation(ucVersion, cV, logger)
-		if err != nil {
+	} else {
```

Comment:
this should probably be `else if ucChannel == cV.Spec.Channel`, so the different-channel, Cincy-configured-off case doesn't attempt to check.  Or the different-channel, Cincy-configured-off case could have an early:

```go
if ucChannel != cV.Spec.Channel {
  if v.Cincinnati {
    return validationPassed, nil
  }
}
```

Or something?

Reply:
Yeah, this actually caused a problem for someone overnight (OHSS-15063) - thanks for spotting it as well. I'll push a fix for this.

Reply:
fix in flight with #341 looks good to me :)